ansible/Dockerfile.Windows2016_Base

```diff
@@ -10,25 +10,16 @@ RUN $ProgressPreference = 'SilentlyContinue'; `
     New-Item -Path C:\ -Name 'temp' -ItemType 'directory'; `
     # cygwin
     [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12; `
-    Invoke-WebRequest 'http://cygwin.com/setup-x86_64.exe' -OutFile C:\temp\cygwin.exe; `
-    Start-Process -Wait -FilePath C:\temp\cygwin.exe -ArgumentList '--quiet-mode --download --local-install --delete-orphans --site https://mirrors.kernel.org/sourceware/cygwin/ --local-package-dir C:\cygwin_packages --root C:\cygwin64'; `
-    Start-Process -Wait -FilePath C:\temp\cygwin.exe -ArgumentList '-q -P wget'; `
-    Start-Process -Wait -FilePath C:\temp\cygwin.exe -ArgumentList '-q -P rsync'; `
-    Start-Process -Wait -FilePath C:\temp\cygwin.exe -ArgumentList '-q -P gnupg'; `
-    Start-Process -Wait -FilePath C:\temp\cygwin.exe -ArgumentList '-q -P git'; `
-    Start-Process -Wait -FilePath C:\temp\cygwin.exe -ArgumentList '-q -P autoconf'; `
-    Start-Process -Wait -FilePath C:\temp\cygwin.exe -ArgumentList '-q -P make'; `
-    Start-Process -Wait -FilePath C:\temp\cygwin.exe -ArgumentList '-q -P unzip'; `
-    Start-Process -Wait -FilePath C:\temp\cygwin.exe -ArgumentList '-q -P zip'; `
-    Start-Process -Wait -FilePath C:\temp\cygwin.exe -ArgumentList '-q -P cpio'; `
-    Start-Process -Wait -FilePath C:\temp\cygwin.exe -ArgumentList '-q -P curl'; `
-    Start-Process -Wait -FilePath C:\temp\cygwin.exe -ArgumentList '-q -P curl-debuginfo'; `
-    Start-Process -Wait -FilePath C:\temp\cygwin.exe -ArgumentList '-q -P libcurl-devel'; `
-    Start-Process -Wait -FilePath C:\temp\cygwin.exe -ArgumentList '-q -P libpng15'; `
-    Start-Process -Wait -FilePath C:\temp\cygwin.exe -ArgumentList '-q -P libpng-devel'; `
-    Start-Process -Wait -FilePath C:\temp\cygwin.exe -ArgumentList '-q -P perl-Text-CSV'; `
-    Invoke-WebRequest 'https://mirrors.kernel.org/sourceware/cygwin/x86_64/release/grep/grep-2.27-2.tar.xz' -OutFile C:\cygwin64\tmp\grep.tar.xz; `
-    Start-Process -Wait -FilePath C:\cygwin64\bin\bash.exe -ArgumentList '-l -c "tar -C / -xvf /tmp/grep.tar.xz"'; `
+    Invoke-WebRequest 'https://ci.adoptopenjdk.net/userContent/winansible/cygwin64.tar.xz' -OutFile C:\temp\cygwin64.tar.xz; `
+    Invoke-WebRequest 'https://ci.adoptopenjdk.net/userContent/winansible/cygwin64/bin/*zip*/bin.zip' -OutFile C:\temp\cyg_toolset.zip; `
+    Expand-Archive -LiteralPath C:\temp\cyg_toolset.zip -DestinationPath C:\temp; `
+    C:\temp\bin\tar --force-local -xvf C:\temp\cygwin64.tar.xz C:\ ; `
+    # Remove speech mark from the %PATH%
+    setx /M PATH "%PATH:\"=%" ; `
+    # Add c:\cygwin64\bin to front of %PATH%
+    setx /M PATH "C:\cygwin64\bin;%PATH%" ; `
```

Comment:
Just to say, I found a different way of doing removing the speech mark with https://github.com/AdoptOpenJDK/openjdk-infrastructure/pull/1495 as I had a lot of trouble with the `setx` syntax in the context of Ansible. If this works, awesome :)